### PR TITLE
Javadocs

### PR DESCRIPTION
**Pull request by ldbonkowski**
I have added javadocs for all the classes (except for the filter pacakage) as well as removed the ItemList class until we determine if it is needed

### CODE REVIEW

**Review by MatthewNeufeld — COMMENTED** [review]
Perfectly descriptive Javadocs ✅

File: app/src/main/java/com/example/househomey/HomeFragment.java [hunk 1]
```diff
@@ -18,37 +19,77 @@
...
     public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
         // Inflate the fragment's layout
         View rootView = inflater.inflate(R.layout.fragment_home, container, false);
 
         itemListView = rootView.findViewById(R.id.item_list);
-        itemAdapter = new ItemAdapter(getContext(), new ItemList(this, itemRef).getItems());
+        itemAdapter = new ItemAdapter(getContext(), itemList);
         itemListView.setAdapter(itemAdapter);
 
+        itemRef.addSnapshotListener((querySnapshots, error) -> {
```
Comment (jdrco) [inline app/src/main/java/com/example/househomey/HomeFragment.java:67]: @ldbonkowski is it cool if we put this in its own method? For example `setupItemListener`. Just to keep things a bit more modular and potentially reusable.

**Review by jdrco — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/househomey/HomeFragment.java (hunk 1 above)
Comment (ldbonkowski) [reply app/src/main/java/com/example/househomey/HomeFragment.java:67]: Yessit, good plan

**Review by MatthewNeufeld — APPROVED** [review]
Looks good

**Review by jdrco — APPROVED** [review]
LGTM

**Pull request merged**